secure_redis/decorators.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from functools import wraps
+from collections import defaultdict
+import logging
+import datetime
+
+from django.conf import settings as global_settings
+import rq.utils
+from rq.compat import string_types
+import django_rq
+from django_rq.queues import get_queue
+from rq.defaults import DEFAULT_RESULT_TTL
+from rq.queue import Queue
+from . import settings
+
+logger = logging.getLogger(__name__)
+
+
+_serializer = None
+
+
+def get_serializer():
```

Comment:
@GhalebKhaled Can you please try to call this as a singular? Like `serializer=get_serializer()` and then you can use `serializer` when ever you need it.